src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -202,12 +273,16 @@ private void RemoveLRUs()
         }
 
         /// <summary>
-        /// The timer callback that adds a request to remove expired items from the event queue.
+        /// This is the method that determines the end time for the event queue task.
```

Comment:
Above on line 271, could we write:
```
_doubleLinkedList.RemoveLast();
```

Reply:
And should the cleanUpIntervalInSeconds be renamed to removeExpiredValuesIntervalInSeconds as we discussed (line 144)?

Reply:
I believe the design decision to use Remove() rather than RemoveLast() was done to avoid multithreading issues where the list is somehow modified by another thread (not sure whether this is possible actually).

@dannybtsai That change should already be made. Can you not see it?